ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
I wonder if Qt was built with a different compiler or with flags that change vtable format.

Is Scan Tailor from ports affected? If not, check what kind of flags or patches they are applying.